ntbs-service/Pages/ServiceDirectory/Region.cshtml

```diff
@@ -6,23 +6,25 @@
 }
 
 <nhs-width-container container-width="Standard">
-    <h2 class="nhsuk-heading-l">@Model.Phec?.Name</h2>
-    <h5 class="nhsuk-heading-xs">Service directory</h5>
+    <h1 class="nhsuk-heading-xl">@Model.Phec.Name</h1>
+    <h2 class="nhsuk-heading-l">Service directory</h2>
 
     <div class="govuk-accordion tb-services-and-case-managers" data-module="govuk-accordion" id="accordion-default">
-        @foreach (var tbService in Model.TbServices)
+        @foreach (var tbServiceWithCaseManagers in Model.TbServicesWithCaseManagers)
         {
+            var tbService = tbServiceWithCaseManagers.Key;
+            var caseManagers = tbServiceWithCaseManagers.Value;
             <div class="govuk-accordion__section ">
                 <div class="govuk-accordion__section-header">
-                    <h2 class="govuk-accordion__section-heading">
+                    <div class="govuk-accordion__section-heading">
```

Comment:
Any reason why this is not an `h` anymore? I thought an `h3` would have been perfect to help navigate the multiple tables :)